doc/actor.md

```diff
@@ -7,570 +8,521 @@ http://stixproject.github.io/data-model/1.2/ta/ThreatActorType/
 
 * This entry is required
 
-### Keyword Key
+* Keyword Key
+  * Plumatic Schema: :id
 
-* Plumatic Schema: :id
-
-### String Value
-
-The URI of this entity.
-
-* Plumatic Schema: Str
+* String Value
+  * The URI of this entity.
+  * Plumatic Schema: Str
 
 ## MapEntry: :type -> String
```

Comment:
I'm not fond of using "->" here at is somehow used for functions.

https://github.com/polygloton/ctim/blob/c9e8e932cea244abd9505f9773f6317dac09d3b7/src/flanders/markdown.cljc#L64

As we somehow anotate some "type" I would rather prefer to use `:` or maybe `::`.